app/models/fiber_channel_address.rb

```diff
@@ -0,0 +1,5 @@
+class FiberChannelAddress < SanAddress
+  def get_address_info
+    [_("wwpn"), wwpn]
+  end
```

Comment:
So it's a flat list with even indexes being names and odd indexes being values like so  `[name, value, name, value, ...]`? Let's make it a list of lists, like so `[[name, value], [name, value], ...]`. This will make it easier to use.
Or are you planning to only have a single name-value pair returned?

Reply:
It suppose to be a single name-value pair returned. This is the format because it being used as his by TextualMultilabel inside the textual_summary page:
https://github.com/ManageIQ/manageiq-ui-classic/pull/7529/files#diff-9ce8d4d0d8c278b02b2e0b6723c908c0342026485b4ded7caf5a7ab3a87c02b3R24

I'm not sure we need a getter for all addresses, but if so maybe we can do get_all_addresses_info like you suggested above.